cluster-provision/images/vm-image-builder/README.md

```diff
@@ -40,16 +43,30 @@ then publish it to a registry. In this case a local registry:
 publish-containerdisk.sh example localhost:1234/myimage:mytag
 ```
 
+To build `example` for arm64(aarch64), only need to set the environmental
+variable.
+```
+export ARCHITECTURE=arm64
+```
+
+To build the Virtual Machine without console output, only need to set the
+environmental variable WITHCONSOLE. This is useful when the building script
+run in a CICD pipeline.
+```
+export WITHCONSOLE=no
```

Comment:
This documents `WITHCONSOLE` but AFAICT the actual variable looked at is `CONSOLE`.